kolibri/core/auth/test/sync_utils.py

```diff
@@ -16,6 +16,9 @@
 from morango.models.core import DatabaseIDModel
```

Comment:
This file contains some utilities to support the syncing tests. I added a few helper methods on the server class to reduce boilerplate cruft in the test file.